test/e2e/installplan_e2e_test.go

```diff
@@ -3895,18 +3716,6 @@ var _ = Describe("Install Plan", func() {
 		_, err = crc.OperatorsV1().OperatorGroups(ns.GetName()).Create(context.TODO(), og, metav1.CreateOptions{})
 		Expect(err).ToNot(HaveOccurred())
 
-		// Wait for the OperatorGroup to be synced and have a status.ServiceAccountRef
-		// before moving on. Otherwise the catalog operator treats it as an invalid OperatorGroup
-		// and fails the InstallPlan
-		Eventually(func() (*corev1.ObjectReference, error) {
```

Comment:
Won't removing these make test flakes more likely? My understanding was that InstallPlan reconciliation will retry 8 times in the span of about one second, and if the OperatorGroup hasn't been reconciled by the eighth try, the InstallPlan reconciliation won't be attempted again for as long as 6 minutes. So anything that ties up olm-operator for 1 second could cause a timeout in a test that waits for an InstallPlan to execute.

Reply:
> Won't removing these make test flakes more likely?

I can see how that could happen. I was mainly looking at it as a proof that we don't necessarily have to wait around for OG reconciliation for the IP to reach it's `Complete` phase, but I guess since other tests prove that OG reconciliation error keeps the IPs in the `Installing` phase, it's safer to poll for successful OG reconciliation to reduce test flakes.